CSLabs.Api/appsettings.Example.json

```diff
@@ -1,10 +1,12 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Server=localhost;Database=cslabs_backend;User=root;Password=;"
+    "DefaultConnection": "Server=localhost;Database=cslabs_backend;User=root;Password=;",
+    "HangfireConnection": "Server=localhost;Database=cslabs_backend;Integrated Security=SSPI;"
```

Comment:
I think we have to create a database specificly for hangfire, however I do not know how this will work with efcore.

Reply:
Is Hangfire using ef core? If not then it won't matter.

Reply:
@jasekiw  it does not from what I see but it looks like it will create all the tables it needs from a schema. I assume this is not an issue being in the same database.

Reply:
Then in that case, no additional connection strings are needed.

Reply:
@Cliftonz Then in that case, no additional connection strings are needed.